src/commands/load.js

```diff
@@ -0,0 +1,33 @@
+let path = require('path');
+
+module.exports = () => {
+	return {
+		run: function(args, api, event) {
```

Comment:
All right, I will add another commit which will fix spacing, increment version number.
In that commit:
* It doesn't throw an exception when you try to `unload' a module that has already been unloaded or doesn't exist
* It doesn't let you load a module that is already loaded
* It still throws an exception when you try to load non-existing module (exception error is `ENOENT: no such file or directory`) - load.js has `try` block, so it seems like it's a problem with `verifyModule` function, I bet it has something like this:
```javascript
            try {
                // do something
            }
            catch (e) {
                console.critical(e);
            }
```
and that's why we see an error in the console. Hmm, maybe it could be fixed within KPM by adding some check (if the dir exists).
I can try to add ability to load modules with `kpm_` prefix without needing to type that prefix, but I don't think that's really needed, it's more for manual modules loading, if someone's doing some testing on KPM module, he can write kpm_modulename then.
Another issue - when I restart Concierge, it rewrites language files, and after that rewrite, all Polish chars are broken... (it doesn't break the main Concierge translations though, only KPM ones [might do that for other modules too, untested]). Maybe it's bad coding (not UTF-8?) once again, but it displays OK, until restart when Concierge rewrites the file.